addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -465,6 +464,29 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasDescriptionArea() {
+            return Boolean(
+                this.thread &&
+                (this.thread.description ||
+                this.thread.isChannelDescriptionChangeable)
+            );
+        }
+
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsDescriptionEditableByCurrentUser() {
+            return Boolean(
+                this.messaging.currentUser &&
+                this.messaging.currentUser.isInternalUser
+            );
+        }
```

Comment:
Thank you for the update.

This should be on thread, not on topbar, and it should take `isChannelDescriptionChangeable` into account (it's not editable by the current user if it's not editable at all).